RELEASE_GUIDE.md

```diff
@@ -1,26 +1,49 @@
 # Release Guide (for maintainers)
 
+## Prerequisites
+
+- You will need a [Hackage](https://hackage.haskell.org/) account that has been invited to be a maintainer of the `purescript` package on Hackage. If you don't have one, create one and ask to be invited as a maintainer.
+- You will need an [NPM](https://www.npmjs.com/) account that has been invited to be a maintainer of the `purescript` package on NPM. If you don't have one, create one and ask to be invited as a maintainer.
+
 ## Before making a release
 
 - Check that there are no unintended breaking changes by compiling [the latest package set](https://github.com/purescript/package-sets/releases/latest)
-- Check that INSTALL.md is up-to-date
-- Regenerate LICENSE: `make license-generator` (see `license-generator/` for
-  details)
-- Write release notes
 
-Additionally, if there are any breaking changes, there are number of downstream
-projects who we should probably at least notify:
+```bash
+stack build
+mkdir wPackageSet
+pushd wPackageSet
+spago init
+spago upgrade-set
+# install all packages in the set
+spago install $(spago ls packages | cut -f 1 -d ' ' | tr '\n' ' ')
```

Comment:
We really need a `spago install --whole-set` 😄

Reply:
Is this just `spago verify-set`?

Reply:
I think it would replace the `spago install` and `spago build` command. However, I am not sure whether it also builds docs. 

@f-f, could you clarify?

Reply:
The question "can I install the entirety of the set?" seems to come up often, and I don't have a good answer for that right now (it will improve once we get the new config format for Spago).

In this thread however, I think Harry is right: installing the whole set and then building it is _basically_ `verify-set` (which does some more stuff, so it's going to be slightly slower than just installing everything and building it, but it's fine)